scripts.js

```diff
@@ -92,15 +92,15 @@ const data = [
 // Array.prototype.filter()
 // 1. Filter the list of inventors for those who were born in the 1500's
 const filterYear = () => {
```

Comment:
No bro, the name of this function

Reply:
Should be `bornInThe1500s` or similar. `filterYear` doesn't make a whole lot of sense on it's own. Nah mean